ansible/roles_ocp_workloads/ocp4_workload_apicurio_registry_operator /tasks/workload.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create OpenShift Apicurio Project
+  k8s:
```

Comment:
Is this related to the variable to "set up users" ? If so it probably needs a when clause.

Reply:
I'm not setting up users. I'm creating a project per users.